runtime/nls/cfre/cfrerr.nls

```diff
@@ -1481,14 +1481,6 @@ J9NLS_CFR_ERR_MULTIPLE_RECORD_ATTRIBUTES.system_action=The JVM will throw a veri
 J9NLS_CFR_ERR_MULTIPLE_RECORD_ATTRIBUTES.user_response=Contact the provider of the classfile for a corrected version.
 # END NON-TRANSLATABLE
 
-# Record is not translatable
-J9NLS_CFR_RECORD_CLASS_CANNOT_BE_ABSTRACT=Record classes cannot be abstract
```

Comment:
Old NLS messages can't be removed, only marked as unused. 
https://github.com/eclipse/openj9/blob/master/runtime/nls/cfre/cfrerr.nls#L27-L29

Reply:
Fixed.